src/Celula.cpp

```diff
@@ -26,15 +26,14 @@ void Celula::nacer(float factorNacimientoParcela, const RGB& RGBParcela){
 	this->RGBCelula = RGBParcela;
 }
 
-void Celula::nacer(float factorNacimientoParcela, const RGB& RGBvecino1,
-		const RGB& RGBvecino2, const RGB& RGBvecino3) {
+void Celula::nacer(float factorNacimientoParcela, RGB* RGBvecino1, RGB* RGBvecino2, RGB* RGBvecino3) {
 	this->estadoCelula = VIVA;
 	this->energiaCelula = ENERGIA_MAX * factorNacimientoParcela;
 	RGBCelula.calcularPromedioRGBes(RGBvecino1, RGBvecino2, RGBvecino3);
 }
 
-const RGB& Celula::getRGB() {
-	return this->RGBCelula;
+RGB* Celula::getRGB() {
+	return &(this->RGBCelula);
```

Comment:
Y si cambiamos el atributo RGBCelula para que sea un puntero directamente? y aca hacemos return this->RGBCelula?